src/storage/conversion/IfNeededConverter.ts

```diff
@@ -1,60 +0,0 @@
-import type { Representation } from '../../http/representation/Representation';
```

Comment:
That makes it a semver.major though!

Are we sure this converter cannot be useful in other stacks?

Reply:
IMO it's already a semver.major due to the changes in the TypedRepresentationConverter (removing one function and changing the signature of another).